### PR TITLE
Refactor DataSourceGeneratedDatabaseConfigurationTest.

### PR DESCRIPTION
**Pull request by yx9o**
Refactor DataSourceGeneratedDatabaseConfigurationTest.

### CODE REVIEW

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/impl/DataSourceGeneratedDatabaseConfigurationTest.java [hunk 1]
```diff
@@ -34,32 +26,60 @@
...
-        assertThat(((HikariDataSource) dataSource).getJdbcUrl(), is(url));
-        FixtureRuleConfiguration next =
-                (FixtureRuleConfiguration) databaseConfiguration.getRuleConfigurations().iterator().next();
-        assertThat(next.getName(), is("rule0"));
-        DataSourceProperties properties = databaseConfiguration.getDataSourceProperties().get("dsc");
-        assertThat(properties.getPoolPropertySynonyms().getStandardPropertyKeys().size(), is(6));
-        assertThat(properties.getConnectionPropertySynonyms().getStandardProperties().get("url"), is(url));
-        assertThat(properties.getConnectionPropertySynonyms().getStandardProperties().get("username"), is(username));
-        assertThat(properties.getConnectionPropertySynonyms().getStandardProperties().get("password"), is(password));
+    public void assertDataSource() {
```
Comment (terrymanu) [inline shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/impl/DataSourceGeneratedDatabaseConfigurationTest.java:40]: It is better to rename method as `assertGetDataSources`

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/impl/DataSourceGeneratedDatabaseConfigurationTest.java [hunk 2]
```diff
@@ -34,32 +26,60 @@
...
+        DataSourceGeneratedDatabaseConfiguration databaseConfig = createDataSourceGeneratedDatabaseConfiguration();
+        HikariDataSource hikariDataSource = (HikariDataSource) databaseConfig.getDataSources().get("normal_db");
+        assertThat(hikariDataSource.getJdbcUrl(), is("jdbc:mock://127.0.0.1/normal_db"));
+        assertThat(hikariDataSource.getUsername(), is("root"));
+        assertThat(hikariDataSource.getPassword(), is(""));
+    }
+    
+    @Test
+    public void assertRuleConfiguration() {
```
Comment (terrymanu) [inline shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/impl/DataSourceGeneratedDatabaseConfigurationTest.java:49]: It is better to rename method as `assertGetRuleConfigurations`

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/impl/DataSourceGeneratedDatabaseConfigurationTest.java [hunk 3]
```diff
@@ -34,32 +26,60 @@
...
+        DataSourceGeneratedDatabaseConfiguration databaseConfig = createDataSourceGeneratedDatabaseConfiguration();
+        FixtureRuleConfiguration ruleConfig = (FixtureRuleConfiguration) databaseConfig.getRuleConfigurations().iterator().next();
+        assertThat(ruleConfig.getName(), is("test_rule"));
+    }
+    
+    @Test
+    public void assertDataSourceProperties() {
```
Comment (terrymanu) [inline shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/impl/DataSourceGeneratedDatabaseConfigurationTest.java:56]: It is better to rename method as `assertGetDataSourceProperties`

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/impl/DataSourceGeneratedDatabaseConfigurationTest.java (hunk 3 above)
Comment (yx9o) [reply shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/impl/DataSourceGeneratedDatabaseConfigurationTest.java]: Done, please review again, thank you.

**Review by terrymanu — APPROVED** [review]

**Pull request merged**